### ISSUE TITLE
[Feature Request] Renaming Apps

### ISSUE THREAD

**Issue by Tealdragon204** [mijorus/gearlever#160]
Sounds simple. Probably isn't. Would be nice to have as some of my apps have changed names to things I'm less familiar with or are just downright confusing

Just a nice to have

**Comment by Raindrac** [comment]
I've been doing this by modifying the .desktop file created for the apps I want to rename, but there's a small inconvinence of having to reapply the modification every time I update an appimage. I'd like this feature too, and in the meantime I recommend doing what I'm doing. A bit of work, but it does the trick!

**Comment by Arcitec** [comment]
@mijorus Hi, I ran into this issue. The native system package conflicts with the AppImage and I see two entries with the exact same app name and icon in my menu. Being able to rename the .desktop name of AppImages would be so convenient to clearly separate the two entries. :)

It's for Envision, an app that gets infrequent updates as a native package, so I sometimes use the AppImage instead. I can't uninstall the native package because then I lose a lot of dependencies.

**Comment by hertz98** [comment]
Yes, it would be nice, if I install two version of the same Application, the name will be confusing, instead I can give a better name to distinguish the two versions, while it will not break if I update one.
